### DISCUSSION TITLE
How can i send a email via API?

### DISCUSSION THREAD

**Discussion by dudrt** [emailjs-com/help#105] [Q&A] [answered]
Hi, i am programming using React Native Expo and i cant make a request, i alredy tried fect, axios, nothing work, but when i try the same json on postman, works fine, why?
here's the code:
note: the `data.name` and `data.email` is not null.

 const Data = {
            service_id: 'I CHANGED',
            template_id: 'I CHANGED',
            user_id: 'I CHANGED',
            template_params: {
                'to_name': data.name,
                'reply_to': 'data.email'
            }
        };
axios({
            method: 'POST',
            url: "https://api.emailjs.com/api/v1.0/email/send",
            data: JSON.stringify(Data),
            headers: {
                'Content-Type': 'application/json',
                },
        }).then((resposta) => {
            setCode(true)
        }).catch(function (error) {
            console.log(error);
        })

**Comment by xr0master** [comment]
Have you tried the official package for react native? https://www.npmjs.com/package/@emailjs/react-native
What error are you getting?

**Reply by dudrt** [reply]
yes i already tried, but i want do with API.
The error is 404, but what is weird is when i try using Postman works fine.

**Reply by xr0master** [reply]
It means the user_id is not found. Perhaps you have extra characters like quota, spaces, and so on.

**Reply by zenonn37** [reply]
I have the same problem using fetch with nextjs. user_id is not recognized.

**Reply by xr0master** [reply]
99.9% of users who complain about the key use all sorts of additional logic, such as environment variables, and find the problem there.

**Reply by dudrt** [reply, marked as answer]
hi, I still cant use it via the API, but im using the package and it works well.

If you're seeing this, use **npm i @emailjs/react-native**
